outputs.tf

```diff
@@ -82,3 +82,13 @@ output "private_route_table_ids" {
   value       = aws_route_table.private[*].id
   description = "IDs of the private route tables"
 }
+
+output "subnet_share_id" {
+  value       = var.share_public_subnets || var.share_private_subnets ? module.subnet_sharing[0].id : ""
```

Comment:
```suggestion
  value       = try(module.subnet_sharing[0].id : null)
```

Reply:
I just picked up this pattern from a smart guy, will start using it ;)